database/seed.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-plusplus */
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+mongoose.Promise = require('bluebird');
+const { Price } = require('./models/prices');
+const { Seller } = require('./models/sellers');
```

Comment:
Refer to the model file comments. If you decide to go with my idea, this will need to be corrected.

Reply:
NA